fuse_core/include/fuse_core/throttled_callback.h

```diff
@@ -57,14 +57,15 @@ class ThrottledCallback
   /**
    * @brief Constructor
    *
-   * @param[in] keep_callback   The callback to call when kept, i.e. not dropped. Defaults to nullptr
-   * @param[in] drop_callback   The callback to call when dropped because of the throttling. Defaults to nullptr
-   * @param[in] throttle_period The throttling period duration in seconds. Defaults to 0.0, i.e. no throttling
-   * @param[in] use_wall_time   Whether to use ros::WallTime or not. Defaults to false
+   * @param[in] keep_callback       The callback to call when kept, i.e. not dropped. Defaults to nullptr
+   * @param[in] drop_callback       The callback to call when dropped because of the throttling. Defaults to nullptr
+   * @param[in] throttle_period     The throttling period duration in seconds. Defaults to 0.0, i.e. no throttling
+   * @param[in] use_wall_time       Whether to use ros::WallTime or not. Defaults to false
    */
   ThrottledCallback(Callback&& keep_callback = nullptr,  // NOLINT(whitespace/operators)
                     Callback&& drop_callback = nullptr,  // NOLINT(whitespace/operators)
-                    const ros::Duration& throttle_period = ros::Duration(0.0), const bool use_wall_time = false)
+                    const ros::Duration& throttle_period = ros::Duration(0.0),
+                    const bool use_wall_time = false)
```

Comment:
Why split lines?
```suggestion
                    const ros::Duration& throttle_period = ros::Duration(0.0), const bool use_wall_time = false)
```

I don't really have a strong opinion on this one, but this isn't really required for this change

Reply:
Yeah fair. I probably modified this signature at some point and reverted it back without going back to the old formatting. As for style, I think I just liked the consistency - Pretty sure that clang-format does this where once it starts to split them across lines, each one gets its own line. Can go back to the old style if preferred.

Reply:
I don't have a strong opinion on the style here, just trying to reduce the changes to the scope of the PR motivation

Reply:
Ok understood, I'll revert this to keep the file history clean.